flumine/utils.py

```diff
@@ -131,19 +131,38 @@ def price_ticks_away(price: float, n_ticks: int) -> float:
 
 
 # todo LRU cache?
-def calculate_exposure(mb: list, ml: list) -> float:
+# JH: LRU cache does not work with list inputs as they are unhashable.
+#     So might need to refactor mb and ml into tuples.
+def calculate_matched_exposure(mb: list, ml: list) -> float:
```

Comment:
Returns a `Tuple`

Reply:
The comment contains "returns the tuple (profit_if_win, profit_if_lose)"

Reply:
Ah, are you referring to the ->?